corehq/apps/fixtures/upload/run_upload.py

```diff
@@ -183,8 +183,11 @@ def validate_fixture_upload(workbook):
         tag = table_def.table_id
         fields = table_def.fields
         item_attributes = table_def.item_attributes
-        data_items = list(workbook.get_data_sheet(tag))
-        for sort_key, data_item in enumerate(data_items):
+        try:
+            data_item = iter(workbook.get_data_sheet(tag)).next()
```

Comment:
Yeah, this isn't super obvious, but the only thing being validated is the header format for each of the sheets; however, we don't have direct access to the header row. Instead, the header row has been used to format each data row into this (deserialized) json `data_item`. That means that at this point the only way to get at the header format is through one of the rows—any of the rows. That's why we only need to look at the first one.